### DISCUSSION TITLE
[v2] Wrapping collections doesn't work on `DataCollection`

### DISCUSSION THREAD

**Discussion by alexrififi** [spatie/laravel-data#159] [Q&A] [answered]
Thank you for the long awaited and **great** release of version 2!

Please note that wrapping collections doesn't work on `DataCollection`, but work on `PaginatedDataCollection`.

For testing, an [example from the documentation](https://spatie.be/docs/laravel-data/v2/as-a-resource/wrapping#content-wrapping-collections) was used.

```php
class Controller
{
    public function index()
    {
        return SongData::collection(Song::all())->wrap('data');
    }
}
```

**Comment by rubenvanassche** [comment, marked as answer]
This will be fixed later today with the next release of data.
